### PR TITLE
Update compatibility.md

### PR DESCRIPTION
**Pull request by misobarisic**
Since `xhost` specific errors (not being able to connect to the xserver, the app not exporting) were not only bound to Void Linux, I decided to add a more inclusive message to compatibility.md.
I encountered the issue of not being able to connect to the host xserver from the container on a minimal Arch installation which previously didn't have the need for configuring `xhost`.

### CODE REVIEW

File: docs/compatibility.md [hunk 1]
```diff
@@ -49,10 +49,12 @@ Distrobox has been successfully tested on:
...
 | Windows WSL2 | | __NOTE WSL2 support is preliminary, and there are many bugs present, any help in improving support is appreciated__ <br> Currently you must work around some incompatibility between WSL2 and Podman, namely [THIS](https://github.com/containers/podman/issues/12236). <br>Install into WSL2 any of the supported distributions in this list. <br> Ensure you have an entry in the `fstab` for the `/tmp` folder:<br> `echo 'tmpfs /tmp tmps defaults 0 0' >> /etc/fstab`.<br>Then reboot the WSL machine `wsl --shutdown` <br>Note that `distrobox export` is not supported on WSL2 and will not work. |
 
+If your container is not able to connect to your host xserver, make sure to install `xhost` on the host machine and run `xhost +si:localuser:$USER` or add it to your `~/.xinitrc`.
```
Comment (89luca89) [inline docs/compatibility.md:56]: Should Be **and** add it to xinirc to make it permanent

**Review by 89luca89 — CHANGES_REQUESTED** [review]

**Comment by 89luca89** [comment]
Cool thanks a lot greatly appreciated 😄

File: docs/compatibility.md (hunk 1 above)
Comment (misobarisic) [reply docs/compatibility.md:56]: The goal is to make it run on every x session startup. Editing `.xinitrc` may not affect it if you're, let's say, using KDE Plasma but your `.xinitrc` starts Gnome.

**Comment by misobarisic** [comment]
I've updated it with a better and less ambiguous message.

**Review by 89luca89 — APPROVED** [review]

**Comment by 89luca89** [comment]
Cool stuff thanks!

**Pull request merged**